app/_landing_pages/authentication.yaml

```diff
@@ -0,0 +1,106 @@
+metadata:
+  title: "Authentication"
+  tags:
+    - authentication
+content:
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h1
+              text: "Authentication"
+              align: center
+
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "What is authentication?"
+              blocks:
+                - type: text
+                  text: |
+                    Authentication is the process of verifying that a requester has permissions to access a resource. 
+                    As its name implies, API gateway authentication authenticates the flow of data to and from your upstream services. 
+                    
+                    Kong Gateway has a library of plugins that support the most widely used methods of API gateway authentication.
+                    
+                    Common authentication methods include:
+
+                    - Key Authentication
```

Comment:
I think it would be good to expand a bit more about authentication methods, this is the page where they are introduced to `Authentication` and we guide them towards different solutions.

We could provide the user a brief intro to the methods `SSO`, `OICD`, `LDAP`, `Basic Auth`, `Key Auth` etc. and even have specific pages for each of those where they can learn more about them and be linked to more specific tutorials, etc

Reply:
By specific page, so you mean a plugin overview page for each authentication plugin? Or something else? And do we want to do these for the MVP?

Reply:
I wasn't thinking about the plugins, but more about the methods. In particular SSO, OICD, LDAP, SAML.